client/public/static/js/account/update_recipe.js

```diff
@@ -0,0 +1,153 @@
+$(document).ready(function () {
+  $("#updateRecipeForm").on("submit", async function (event) {
+    event.preventDefault();
+    const username = utils.getUserNameFromCookie();
+    const userId = await utils.getUserIdFromUsername(username);
+    let formRecipeName = document.getElementById('recipeName');
+    
+    let urlAction = "";
+    let request = {};
+    let successMessage = "";
+    let errorMessage = "";
+
+    urlAction = DELETE_ACTION;
+    request = {
+      recipeName: formRecipeName.value
+    }
+    successMessage = SUCCESSFULLY_DELETED_RECIPE;
+    errorMessage = UNABLE_TO_DELETE_RECIPE_ERROR;
+    
+    let url = `${USER_FAVORITES_RECIPES_CRUD_URL}/${userId}/recipe/delete_recipe`;
+    console.log(`Sending [${urlAction}] request to: ${url}`)
+
+    fetch(url, {
+      method: urlAction,
+      headers: {
+        'Content-Type': DEFAULT_DATA_TYPE
+      },
+      body: JSON.stringify({ favorites: request })
+    }).then(response => {
+      if (!response.ok) {
+        throw new Error(errorMessage);
+      }
+    }).then(data => {
+      if (data.message === successMessage) {
+        console.log(successMessage);
+        utils.setStorage("deleteRecipeMessage", successMessage);
+      }
+    }).catch(error => {
+      console.log(error);
+      utils.showAjaxAlert("Error", error.message);
+    });
+
+    if (!username) {
+      console.error(UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+      utils.showAjaxAlert("Error", UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+      return;
+    }
+
+    if (!userId) {
+      console.error(UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+      utils.showAjaxAlert("Error", UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+      return;
+    }
+
+    const formData = new FormData(this);
+    formData.append('userCreated', true);
+    console.log("FORM DATA: " + formData);
+
+    url = `${USER_FAVORITES_RECIPES_CRUD_URL}/${userId}/recipe/create_recipe`;
+    console.log(`Sending request to: ${url}`);
+
+    fetch(url, {
+      method: POST_ACTION,
+      body: formData,
+    }).then(response => {
+      if (response.ok) {
+        console.log(SUCCESSFULLY_UPDATED_RECIPE);
+        utils.setStorage("createRecipeMessage", SUCCESSFULLY_UPDATED_RECIPE);
+        window.location = MY_RECIPES_ROUTE;
+      } else {
+        throw new Error(UNABLE_TO_UPDATE_RECIPE_UNEXPECTED_ERROR);
+      }
+    }).catch(error => {
+      console.log(error);
+      utils.showAjaxAlert("Error", error.message);
+    });
+  });
+});
+
+function RecipeDetailsView() {
```

Comment:
Let's rename this to user recipe details view or something that aligns with what the function is updating